recipes/mailpit/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Build the web UI
+npm install
+npm run build
+
+# Disable CGO to avoid Security Framework API compatibility issues with macOS 10.13 SDK
+# (SecTrustCopyCertificateChain and SecTrustEvaluateWithError require macOS 10.14+)
+export CGO_ENABLED=0
```

Comment:
```suggestion
```
Rather than disabling CGO, you should just create a `conda_build_config.yaml` file with these contents:
```
MACOSX_SDK_VERSION:  # [osx and x86_64]
  - 10.14            # [osx and x86_64]